src/policy/truc_policy.h

```diff
@@ -15,11 +15,12 @@
 #include <set>
 #include <string>
 
```

Comment:
In f5b26e5b9be41cdab6e75aa0a3fbbb765a96ed06

This left out v3 mentions in the unit tests, maybe just leave out just the variable names as you stated from PR description.